VK_App/VK_App/View/UserImageViewCell.swift

```diff
@@ -11,13 +11,7 @@ final class UserImageViewCell: UICollectionViewCell {
 
     // MARK: - Public Methods
 
-    func setupInfo(_ imageName: User?) {
-        setupUIInfo(imageName)
-    }
-
-    // MARK: - Private Methods
-
-    private func setupUIInfo(_ imageName: User?) {
-        userImageView.image = UIImage(named: imageName?.userImageName ?? "")
+    func setupInfo(_ imageName: String?) {
```

Comment:
```suggestion
    func configure(_ imageName: String?) {
```
всегда configure никаких setupInfo